app.ts

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
```

Comment:
In typescript you should really use import instead of require so these lines would be:

```
import express from 'express';
import nunjucks from 'nunjucks';
import path from 'path';
```